### ISSUE TITLE
Cannot download file with % or # or ' in the filename

### ISSUE THREAD

**Issue by blokeley** [blokeley/backup_dropbox#3]
```
Traceback (most recent call last):
  File "backup.py", line 257, in save_file
    fout.write(download_file(headers, member_id, metadata['path']))
  File "backup.py", line 229, in download_file
    r.raise_for_status()
  File "C:\Users\Tom Oakley\Anaconda3\lib\site-packages\requests\models.py", line 837, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 400 Client Error: Bad Request for url: 
```

Test with parentheses too.

Propose 
`path.replace(r'%', r'%25')` 

or 
`urllib.parse.quote(path)`

**Comment by blokeley** [comment]
The characters appear to be causing problems because they are special characters in URLs.  The switch to using the dropbox SDK rather than the requests library in backup_dropbox v2 may fix these issues.

**Comment by blokeley** [comment]
Fixed in v2.0+
